wp-content/themes/pub/wporg-learn-2020/css/components/_section-heading.scss

```
@@ -31,7 +36,24 @@
 		}
 	}
 
+	.button.button-large {
+		border: 1px solid #E2E4E7;
+		border-radius: 4px;
+		background-color: #F8F9F9;
+		padding-top: 12px;
+		padding-bottom: 12px;
+		height: 42px;
+		font-style: normal;
+		font-weight: 600;
+		font-size: 13px;
+		line-height: 18px;
+	}
+
 	&--with-space {
 		padding-bottom: 16px;
+
+		&.large-space {
+			padding-bottom: 28px;
+		}
```

From a design consistency standpoint I think this should be applied sitewide via the existing class applied to these headings:
```suggestion
	&--with-space {
		padding-bottom: 28px;
```